en, zh: update the doc of configuring tidb cluster (#1288)
pingcap/docs-tidb-operator: en/configure-a-tidb-cluster.md

#### Storage Class

> You can set the storage class by modifying `storageClassName` of each component in `${cluster_name}/tidb-cluster.yaml` and `${cluster_name}/tidb-monitor.yaml`. For the [storage classes]((https://kubernetes.io/zh/docs/concepts/storage/storage-classes/) supported by the Kubernetes cluster, check with your system administrator.

Reviewer: ```suggestion
You can set the storage class by modifying `storageClassName` of each component in `${cluster_name}/tidb-cluster.yaml` and `${cluster_name}/tidb-monitor.yaml`. For the [storage classes](https://kubernetes.io/docs/concepts/storage/storage-classes/) supported by the Kubernetes cluster, check with your system administrator.
```

This section introduces how to configure the parameters of TiDB/TiKV/PD/TiFlash/TiCDC.

#### Configure TiDB parameters

TiDB parameters can be configured by `spec.tidb.config` in TidbCluster Custom Resource.

For example:

> For TiDB Operator v1.1.6 and later versions, please use TOML format configuration:

Reviewer 2: ```suggestion
For TiDB Operator v1.1.6 and later versions, configure the parameters in the TOML format as follows:
```

    replicas: 1
    service:
      type: ClusterIP
    config: |
      split-table = true
      oom-action = "log"
    requests:
      cpu: 1
```

> For TiDB Operator versions prior to v1.1.6, please use the YAML format configuration:

Reviewer 2: ```suggestion
For TiDB Operator versions earlier than v1.1.6, configure the parameters in the YAML format as follows:
```

For example:

> For TiDB Operator v1.1.6 and later versions, please use TOML format configuration:

Reviewer 2: ```suggestion
For TiDB Operator v1.1.6 and later versions, configure the parameters in the TOML format as follows:
```

spec:
....
  tikv:
    image: pingcap/tikv:v5.0.1
    config: |
      [storage]
        [storage.block-cache]
          capacity = "16GB"
    replicas: 1
    requests:
      cpu: 2
```

> For TiDB Operator versions prior to v1.1.6, please use the YAML format configuration:

Reviewer 2: ```suggestion
For TiDB Operator versions earlier than v1.1.6, configure the parameters in the YAML format as follows:
```

For example:

> For TiDB Operator v1.1.6 and later versions, please use TOML format configuration:

Reviewer 2: ```suggestion
For TiDB Operator v1.1.6 and later versions, configure the parameters in the TOML format as follows:
```

spec:
.....
  pd:
    image: pingcap/pd:v5.0.1
    config: |
      lease = 3
      enable-prevote = true
```

> For TiDB Operator versions prior to v1.1.6, please use the YAML format configuration:

Reviewer 2: ```suggestion
For TiDB Operator versions earlier than v1.1.6, configure the parameters in the YAML format as follows:
```

For example:

> For TiDB Operator v1.1.6 and later versions, please use TOML format configuration:

Reviewer 2: ```suggestion
For TiDB Operator v1.1.6 and later versions, configure the parameters in the TOML format as follows:
```

  ...
  tiflash:
    config:
      config: |
        [flash]
          [flash.flash_cluster]
            log = "/data0/logs/flash_cluster_manager.log"
        [logger]
          count = 10
          level = "information"
          errorlog = "/data0/logs/error.log"
          log = "/data0/logs/server.log"
```

> For TiDB Operator versions prior to v1.1.6, please use the YAML format configuration:

Reviewer 2: ```suggestion
For TiDB Operator versions earlier than v1.1.6, configure the parameters in the YAML format as follows:
```

For example:

> For TiDB Operator v1.2.0-rc.2 and later versions, please use TOML format configuration:

Reviewer 2: ```suggestion
For TiDB Operator v1.2.0-rc.2 and later versions, configure the parameters in the TOML format as follows:
```

```yaml
apiVersion: pingcap.com/v1alpha1
kind: TidbCluster
metadata:
  name: basic
spec:
  ...
  ticdc:
    config: |
      gc-ttl = 86400
      log-level = "info"
```

> For TiDB Operator versions prior to v1.2.0-rc.2, please use the YAML format configuration:

Reviewer 2: ```suggestion
For TiDB Operator versions earlier than v1.2.0-rc.2, configure the parameters in the YAML format as follows:
```

For example:

> 自 v1.2.0-rc.2 版本起支持透传 TOML 配置给组件:

Reviewer: 噫，这里是中文了

### Configure PV for TiDB slow logs

By default, TiDB Operator creates a `slowlog` volume (which is an `EmptyDir`) to store the slow logs, mounts the `slowlog` volume to `/var/log/tidb`, and prints slow logs in the `stdout` through a sidecar container.

> **Warning:**
>

> > By default, the slow query logs stored using the `EmptyDir` volume will be lost after the Pod is deleted (for example, rolling update). Please make sure that a log collection solution has been deployed in the Kubernetes cluster to collect logs of all containers. If you do not deploy a log collection solution, please **must** follow the below configuration to use a persistent volume to store the slow query logs.

Reviewer 2: ```suggestion
> By default, after a Pod is deleted (for example, rolling update), the slow query logs stored using the `EmptyDir` volume are lost. Make sure that a log collection solution has been deployed in the Kubernetes cluster to collect logs of all containers. If you do not deploy such a log collection solution, you **must** make the following configuration to use a persistent volume to store the slow query logs.
```

> If you want to use a separate PV to store the slow logs, you can specify the name of the PV by configuring `spec.tidb.slowLogVolumeName` and configure the PV in `spec.tidb.storageVolumes` or `spec.tidb.additionalVolumes`.

Reviewer 2: ```suggestion
If you want to use a separate PV to store the slow logs, you can specify the name of the PV in `spec.tidb.slowLogVolumeName`, and then configure the PV in `spec.tidb.storageVolumes` or `spec.tidb.additionalVolumes`.
```
